quality/postman/emodb_tests.postman_collection.json

```diff
@@ -38643,15 +38640,9 @@
 															"});",
 															"",
 															"var jsonData = pm.response.json();",
-															"",
-															"pm.test(\"Number of returned documents\", async function () {",
```

Comment:
why this was removed?

Reply:
removed from being unnecessary, since it was a static check